.github/workflows/staging-test-tar-arm64.yml

```diff
@@ -0,0 +1,287 @@
+name: Process TAR Artifacts
+
+on:
+  # schedule:
+  #   - cron: '0 10 * * *'
+  repository_dispatch:
+    types: [staging-build-tar]
```

Comment:
change to `staging-test-tar-arm64` consistent with workflow name as same as others

Reply:
Changed.

Reply:
Its `staging-test-tar-arm64`

Reply:
Changed it to :staging-test-tar-arm64